packages/rancher-prom2teams/generated-changes/overlay/templates/configmap-pre-install.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $valid := list "DEBUG" "INFO" "WARNING" "ERROR" "CRITICAL" -}}
```

Comment:
Sorry to flip flop, but now that this looks almost exactly the same as ` packages/rancher-prom2teams/generated-changes/exclude/templates/configmap.yaml`, we should have this file just be a patch on configMap.yaml

Reply:
fixed.